app/Http/Controllers/FoodController.php

```diff
@@ -73,4 +75,51 @@ public function store(FoodPostRequest $request)
             return redirect()->back();
         }
     }
+
+    /**
+     * Show the form for editing the specified resource.
+     *
+     * @param int $id of food
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function edit($id)
+    {
+        $food = $this->food->findOrFail($id);
+        $categories = Category::orderBy('id', 'DESC')->get();
+        return view('foods.edit', ['food' => $food, 'categories' => $categories]);
+    }
+
+    /**
+     * Update the specified resource in storage.
+     *
+     * @param \Illuminate\Http\Request $request of foods
+     * @param int                      $id      of food
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function update(FoodPutRequest $request, $id)
+    {
+        
+        $food = $this->food->findOrFail($id);
+        $arrFoods = $request->all;
```

Comment:
$request->all() , 
if `$request->all`, this is attribute of $request, and result is Null.
careful :)